lib/generator.rb

```diff
@@ -11,6 +11,26 @@ def initialize(track:, metadata:)
     end
   end
 
+  # A generator has to have a #call method, that's the only requirement.
+  # It doesn't have to be a RepositoryDelegator
```

Comment:
A generator needs to be `RepositoryDelegator`, but `GenerateAllTests` is a meta generator, so _it_ doesn't need one. I suspect the term "generator" is too overloaded.